src/components/ListHeader.js

```diff
@@ -43,16 +74,13 @@ export const ListHeader = ({ queryVariables, data, updateListData }) => {
 
   return (
     <Wrapper>
-      <DropdownSelect
-        data={dropdownData}
-        setData={setDropdownData}
-        label={texts.categoryFilter.label}
-      />
+      <DropdownSelect data={dropdownData} setData={setDropdownData} label={dropdownLabel} />
```

Comment:
dropdownData is dropdownInitialData, isn't it ? 
Here I expected data={dropdownInitialData}

Reply:
`dropdownData` is the state variable we use for the select. `dropdownInitialData` is a method we use to calculate the dropdown data when the component gets rendered. maybe the naming can be optimized somehow. any suggestions?

Reply:
the dropdown data when the component gets rendered should not go on our stat variable directly ?

Reply:
it does: `dropdownInitialData(query)` is called in the `useState` for `dropdownData` to have it set directly.